gallery/src/PageHeader/HostedVersionsSelect.tsx

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useEffect, ChangeEventHandler, useState } from 'react';
+import { NxFormSelect } from '@sonatype/react-shared-components';
+
+import packageJson from '../../package.json';
+
+import './HostedVersionsSelect.scss';
+
+const CURRENT_VERSION = packageJson.version;
+const HOSTED_VERSIONS_JSON_URL = 'https://gallery.sonatype.dev/hosted-versions.json';
```

Comment:
Not sure how I feel about `gallery.sonatype.dev` being hard-coded.  I wonder if it should either be an environment variable, or if the URL should just have the path part (`/hosted-versions.json`) along with logic to hide the dropdown if the file isn't found.

Reply:
> Not sure how I feel about `gallery.sonatype.dev` being hard-coded. I wonder if it should either be an environment variable, or if the URL should just have the path part (`/hosted-versions.json`) along with logic to hide the dropdown if the file isn't found.

Yup! I updated it here: 18aab428bcb0badf4d66b08b695127dd78b9a43e
Let me know what you think!